src/python/grapl_e2e_tests/tests.py

```diff
@@ -123,3 +149,81 @@ def expected_gql_asset() -> Mapping[str, Any]:
             }
         ],
     }
+
+
+# -----------------------  MODEL PLUGIN HELPERS -------------------------------------------
+
+# TODO: move these into their own file once that's doable with e2e/pants
+
+
+def upload_model_plugin(
+    model_plugin_client: ModelPluginDeployerClient,
+) -> bool:
+    logging.info("Making request to /deploy to upload model plugins")
+
+    plugin_path = "./schemas"
+    jwt = EngagementEdgeClient().get_jwt()
+
+    try:
+        check_plugin_path_has_schemas_file(plugin_path)
+    except:
+        logging.info("Plugin path is does not contain")
+    finally:
```

Comment:
Why does this silently fail? If the `check-plugin-path` fails that implies the rest of things are gonna fail too; I think we should remove the try-except-finally

Reply:
Removed try-catch-finally